### DISCUSSION TITLE
How to disable the Tags pages?

### DISCUSSION THREAD

**Discussion by andrey-jef** [ObsidianPublisher/obsidian-github-publisher#251] [Q&A — Template] [answered]
Hi Mara,

Is there an option (in the Obsidian plugin or the `mkdocs.yml`) to disable the "Tags" page (or at least hide it from the top navigation bar in the published website) created by the [tags customization](https://obsidian-publisher.netlify.app/advanced/customization/#tags) from your template?

I tried to 
- delete the "tags.md" in "docs" folder 
- commented out these 2 lines from my `mkdocs.yml`
```python
  - tags:
      tags_file: tags.md
```
but the "tags.md" still come back when the "update" action is executed.

Thanks

**Comment by Mara-Li** [comment, marked as answer]
Remove the plugin : https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/#built-in-tags-plugin

**Reply by andrey-jef** [reply]
aha. Thanks.

I can disable the built-in `tags` plugin by specifying this in `mkdocs.yml`
```python
plugins:
  - tags:
      enabled: false
```

https://squidfunk.github.io/mkdocs-material/plugins/tags/#config.enabled

After having disabled the plugin, the file "tags.md" does not reappear when "update" action is executed.
